cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_data_resources.go

```diff
@@ -399,3 +401,55 @@ func (s *Server) updatePkgInstall(ctx context.Context, cluster, namespace string
 	}
 	return &pkgInstall, nil
 }
+
+// inspectKappK8sResources returns the list of k8s resources matching the given listOptions
+func (s *Server) inspectKappK8sResources(ctx context.Context, cluster, namespace, packageId string) ([]*corev1.ResourceRef, error) {
```

Comment:
This is mostly the code that kapp CLI itself uses for inspecting an app, but adapted to our needs.